src/app/containers/PodcastExternalLinks/index.jsx

```diff
@@ -114,7 +112,7 @@ const PodcastExternalLinks = ({ brandTitle, links }) => {
           dir={dir}
         >
           <span>
-            <span lang={lang}>{firstLink.linkText}</span>
+            <span lang={firstLink.lang}>{firstLink.linkText}</span>
```

Comment:
Do we handle no lang?

Reply:
I like the idea of handling the language via the language based on some logic. How do you feel about something like `getPodcastExternalLinkLang("Spotify")`?

Reply:
I like the idea of handling the language via the language based on some logic. How do you feel about something like getPodcastExternalLinkLang("Spotify") @jroebu14 ?

Reply:
what about something like

```
const getExternalLinkLang = ({ podcastService,  serviceLang }) => {
  if (['Spotify', 'Apple', 'RSS'].includes(podcastService)) {
    return 'en-GB'
  }
  return serviceLang
}
```

then use it like `getExternalLinkLang({ podcastService: 'Spotify', serviceLang: lang })`

Reply:
@jroebu14 that's a much better solution. I've updated the PR. Now we are using a `getExternalLinkLang` utility library